<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Username">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browseProductsButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse Products</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addToCartButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to Cart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewCartButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>View Cart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkoutButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Checkout </string>
    </property>
   </widget>
   <widget class="QPushButton" name="payButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Pay </string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewOrdersButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>View Orders </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="productIdLineEdit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="quantityLineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Username_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Product ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="Username_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantity</string>
    </property>
   </widget>
   <widget class="QListWidget" name="messagesListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>781</width>
      <height>421</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFromCartButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove from Cart</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menueCommerce_Client">
    <property name="title">
     <string>eCommerce Client</string>
    </property>
   </widget>
   <addaction name="menueCommerce_Client"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
